ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -1787,12 +2013,11 @@ ReadOverlayArray:
     RTL
 }
 
+; TODO: These comparison values will need to be calulated somehow or set
+; depending on the area. Right now they are hardcoded to work with the
+; pyramid area.
```

Comment:
typo: calculated 

Also I do see potential here for another system to allow for more control over the background overlays based on the area

Reply:
Nice catch, and yeah there is some potential here for sure. I just don't understand all of the math that is going on here quite yet enough to do it. It was a pain in the butt when I had to adjust it for AHE.